### ISSUE TITLE
java.lang.ClassNotFoundException

### ISSUE THREAD

**Issue by shyankirat** [thiloklein/matchingMarkets#7]
Error in .jnew("smi") : java.lang.ClassNotFoundException
Called from: .jnew("smi")

I get the above mentioned error when I am trying to run the hri program with any condition. Is there any suggestion for it?

**Comment by thiloklein** [comment]
The `.jnew` creates a new Java class using R package `rJava`. It seems that the error can be resolved by installing a Java version (i.e. 64-bit Java or 32-bit Java) that fits to the type of R version that you are using (i.e. 64-bit R or 32-bit R). See the note on Java in the installation section at [matchingMarkets.org](http://matchingMarkets.org). Let me know if this helps.

**Comment by shyankirat** [comment]
I checked as you have suggested and everything is there. Still, the same problem is being displayed. Do I need to make the class file of the java programs? I tried that too but is also not happening due to error. The method seems fine as I tried another simple java code to run it worked which is as follows,
library(rJava)
.jinit('.')
s <- .jnew('java/lang/String', 'Helloworld!Thisisshyan')
o<-.jcall(s, 'I', 'length')
print("Output")
print(o)

Judging the error I was thinking of compiling and getting the class file which I could not do. Repeatedly I am getting the following error
Error in .jnew("smi") : java.lang.ClassNotFoundException
Called from: .jnew("smi")
